packages/ketcher-core/src/application/render/restruct/retext.ts

```diff
@@ -74,6 +74,11 @@ class ReText extends ReObject {
     return refPoints
   }
 
+  getVBoxObj(): Box2Abs {
+    const [p0, _, p2] = this.getReferencePoints()
```

Comment:
Could you, please, clarify what are the `p0` and `p2`?

Reply:
fixed